### ISSUE TITLE
Vi2En:when training is over, the bleu is still 6.11, I find that BLEU  increases very slowly.

### ISSUE THREAD

**Issue by Artificial2idiot** [tensorflow/nmt#276]
I have download data correctly through `nmt/scripts/download_iwslt15.sh /tmp/nmt_data`.
And then i use
 `python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --learning_rate=1.0 \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu` 
to train this model.
However, when it's done after 12000 steps, BLEU is still very small, only 6.11. I can find that Bleu increases very very slowly, I don't know why? if anyone know, please give me a hand, thankyou.

**Comment by smallhaolzh** [comment]
I have met the same problems, and I got 5.+ blue. Has you solved that?
